pkg/render/render.go

```diff
@@ -163,6 +163,12 @@ func RenderDir(archiveDir string, a *apptypes.App, downstreams []downstreamtypes
 		NoProxyEnvValue:    os.Getenv("NO_PROXY"),
 	}
 
+	if createNewVersion {
```

Comment:
does this fix a bug as well? the variable name threw me off

Reply:
I don't think this had any side effects until now.